tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+import os
+from wptserve import server
+
+import pytest
+
+WDIR = 'data'
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+@pytest.fixture
+def setUp():
+    httpd = server.WebTestHttpd(port=8080,
+                                        doc_root=os.path.join(HERE, WDIR),
+                                        host='127.0.0.1')
+    wdir = 'http://{}:{}'.format(httpd.host, httpd.port)
```

Comment:
Instead of continuing in using our custom `wdir` variable I would suggest we switch to make use of the public method [get_url()](https://dxr.mozilla.org/mozilla-central/rev/c2593a3058afdfeaac5c990e18794ee8257afe99/testing/web-platform/tests/tools/wptserve/wptserve/server.py#709-715) inside the tests.

That way we also save us from not having the fixture to return an additional value.

Reply:
You didn't address that comment correctly. We don't want a global `wdir` variable.

Reply:
In my tests. How am I supposed to use get_url() on httpd as it is a function. Do I need to make a httpd variable in my tests as well?

Reply:
Oh, I missed that you wrongly return the return value of `httpd.start()`. Instead you have to return `httpd`. That way you have access to the method.